bench/gemm.h

```diff
@@ -676,11 +760,31 @@ static void SRCNN935GemmArguments(benchmark::internal::Benchmark* b) {
 }
 
 // SRCNN (9-5-5)
-static void SRCNN955GemmArguments(benchmark::internal::Benchmark* b) {
+static void SRCNN955GemmArguments(BenchmarkWrapper* b) {
   b->ArgNames({"M", "N", "K"});
 
   /*           M       N       K    */
   b->Args({376 * 376, 64,  1 * 9 * 9});
   b->Args({372 * 372, 32, 64 * 5 * 5});
   b->Args({368 * 368,  1, 32 * 5 * 5});
 }
+
+// Large Language Model (Generic)
+static void LLMGemmArguments(BenchmarkWrapper* b) {
+  b->ArgNames({"M", "N", "K", "Bl"});
```

Comment:
Can we increase M to something like 128? For MR = 1 kernels, M > 1 means more invocations, but for MR > 1 it might not be representative.